.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: mbgl/ci:r4-linux-clang-3.9
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-mapbox-gl-js-cache
+      - run:
+          name: install dependencies
+          command: |
+            bash ./.circleci/dependencies.sh
+      - save_cache:
+          key: v1-mapbox-gl-js-cache
+          paths:
+            - '~/.yarn'
+            - '.eslintcache'
+            - 'node_modules'
+      - run:
+          name: run tests
+          command: |
+            bash ./.circleci/test.sh
+      - store_artifacts:
+          path: "test/integration/render-tests/index.html"
+      - store_artifacts:
+          path: "test/integration/query-tests/index.html"
+      - run:
+          name: deploy if tagged
+          when: on_success
+          command: |
+            if [[ $CIRCLE_TAG =~ v[0-9]+\.[0-9]+\.[0-9]+(\-dev)? ]]; then
```

Comment:
I think it was a tagged build unless I'm missing something 🤔  I made (and subsequently deleted) a `v0.0.0-dev` tag to test out the conditional.

<img width="604" alt="screen shot 2017-07-06 at 8 55 14 am" src="https://user-images.githubusercontent.com/2425307/27920268-e6c16eb6-6228-11e7-848d-69f42272de57.png">